http-server/src/main/java/io/micronaut/http/server/csp/CspFilter.java

```diff
@@ -0,0 +1,71 @@
+package io.micronaut.http.server.csp;
```

Comment:
Missing a header. Please run `./gradlew project:checkstyleMain` to verify the changes have no code style errors

Reply:
Sorry, missed that. will run checkstyle.